### PR TITLE
Add TUS support information on FileInfo object

### PR DESCRIPTION
**Pull request by PVince81**
Whenever a list() or fileInfo() call response contains TUS headers,
return these through the fileInfo.getTusSupport() method.

If no TUS headers are found, that method returns null.

Fixes https://github.com/owncloud/owncloud-sdk/issues/458

- [x] unit tests
- [x] test against actual backend

### CODE REVIEW

**Comment by PVince81** [comment]
I've re-read the spec and apparently only "Tus-Version" MUST be set and not the others.

- [ ] rely on "Tus-Version" instead of "Tus-Resumable" to detect if TUS is set at all

**Comment by PVince81** [comment]
adjusted, please review

File: tests/filesTest.js [hunk 1]
```diff
@@ -523,4 +525,85 @@ describe('Main: Currently testing files management,', function () {
...
+    afterEach(function () {
+      parseBodyStub.restore()
+      xhr.restore()
+    })
+
+    it('returns TUS support information when TUS headers are set for a list call', function (done) {
+      const promise = oc.files.list('')
+      promise.then(entries => {
+        const tusSupport = entries[0].getTusSupport()
+        expect(tusSupport.resumable).toEqual('1.0.0')
```
Comment (LukasHirt) [inline tests/filesTest.js:554]: > I've re-read the spec and apparently only "Tus-Version" MUST be set and not the others.

Should this line still be here then or removed?

Comment (PVince81) [reply tests/filesTest.js:554]: I think we can keep it as I want to check whether the header is parsed when it's present (the test is only using a mock anyway)

**Pull request merged**